### PR TITLE
Changes to blocks & styling for view & header area - Press Clipping

### PR DESCRIPTION
**Pull request by AdamMcFet**

### CODE REVIEW

File: project/sites/pressclippingsnics/themes/pressclippingsnics_theme/src/scss/components.scss [hunk 1]
```diff
@@ -20,3 +20,22 @@
...
+  h2 {
+    @include adjust-font-size-to(30px);
+  }
+
+  p {
+    @include adjust-font-size-to(16px);
+  }
+}
+#header {
+  background: #f8f8f8;
```
Comment (soda736) [inline project/sites/pressclippingsnics/themes/pressclippingsnics_theme/src/scss/components.scss:37]: Should maybe add these colours as variables?

File: project/sites/pressclippingsnics/themes/pressclippingsnics_theme/src/scss/components.scss [hunk 2]
```diff
@@ -20,3 +20,22 @@
...
+  border: solid 1px #e7e7e7;
+  border-radius: 6px;
+  padding: 5px 5px 0px 20px;
```
Comment (soda736) [inline project/sites/pressclippingsnics/themes/pressclippingsnics_theme/src/scss/components.scss:40]: If a value is 0 it doesn't need px so should be `padding: 5px 5px 0 20px;`
Might actually be better as rem too `padding: .5rem .5rem 0 2rem;`?

**Review by soda736 — CHANGES_REQUESTED** [review]

File: project/sites/pressclippingsnics/themes/pressclippingsnics_theme/src/scss/components.scss (hunk 1 above)
Comment (AdamMcFet) [reply project/sites/pressclippingsnics/themes/pressclippingsnics_theme/src/scss/components.scss]: Done thanks Adam

File: project/sites/pressclippingsnics/themes/pressclippingsnics_theme/src/scss/components.scss (hunk 2 above)
Comment (AdamMcFet) [reply project/sites/pressclippingsnics/themes/pressclippingsnics_theme/src/scss/components.scss]: Done thanks Adam

**Review by soda736 — APPROVED** [review]

**Pull request merged**